components/io_expander/esp_io_expander_ht8574/esp_io_expander_ht8574.c

```diff
@@ -1,23 +1,22 @@
 /*
- * SPDX-FileCopyrightText: 2023 Espressif Systems (Shanghai) CO LTD
+ * SPDX-FileCopyrightText: 2023-2024 Espressif Systems (Shanghai) CO LTD
  *
  * SPDX-License-Identifier: Apache-2.0
  */
 
 #include <inttypes.h>
 #include <string.h>
 #include <stdlib.h>
-
 #include "driver/i2c.h"
```

Comment:
I'm sorry, one last thing. Please remove this deprecated header

Reply:
Sorry for overlooking this, it’s now fixed.